src/components/Profile/EditProfile.jsx

```diff
@@ -28,6 +29,22 @@ const EditProfile = ({userMode, setUserMode}) => {
   const handleOpenTokenModal=()=>{
     setTokenModalOpen(true);
   }
+  useEffect(() => emailjs.init("orjxh6oY0HnuTQT2f"), []);
```

Comment:
Can we move this token to Vercel APP environment configuration, and pick this from the config?